src/Bridge/Doctrine/Orm/Extension/EagerLoadingExtension.php

```diff
@@ -198,6 +198,16 @@ private function addSelect(QueryBuilder $queryBuilder, string $entity, string $a
                 if (true === $targetClassMetadata->hasField($property) && (true === $propertyMetadata->getAttribute('fetchable') || true === $propertyMetadata->isReadable())) {
```

Comment:
@meyerbaptiste so should I also remove the strict check against true here alos, because they are rather the same line. I can update both at once